### PR TITLE
Add more actions needed for storage test

### PR DESCRIPTION
**Pull request by MidhunSureshR**
See https://github.com/matrix-org/trafficlight/pull/43 for trafficlight side of things

### CODE REVIEW

File: cypress/e2e/trafficlight/trafficlight.spec.ts [hunk 1]
```diff
@@ -25,6 +25,10 @@ type JSONValue =
     | { [x: string]: JSONValue }
     | Array<JSONValue>;
 
+Cypress.on('uncaught:exception', (err, runnable) => {
```
Comment (michaelkaye) [inline cypress/e2e/trafficlight/trafficlight.spec.ts:28]: Per : https://github.com/matrix-org/trafficlight-adapter-element-web/pull/4#issuecomment-1270048846

I'm not sure if we want to do this; last time it came up it was a real issue in element web that should be caught.

Comment (michaelkaye) [reply cypress/e2e/trafficlight/trafficlight.spec.ts:28]: Though when i was doing it myself in the other PR; i was thinking along the lines that it doesn't matter so long as the end user experience is OK within the context of the test... so ... not sure.

Odd that this has popped up twice in two weeks though.

**Comment by MidhunSureshR** [comment]
@michaelkaye All web based clients should be able to support these actions.

File: cypress/e2e/trafficlight/trafficlight.spec.ts (hunk 1 above)
Comment (michaelkaye) [reply cypress/e2e/trafficlight/trafficlight.spec.ts:28]: Yeah, let's actually put it in then.

**Comment by michaelkaye** [comment]
(We're good to merge)

**Pull request merged**